vdr/vdr.go

```diff
@@ -167,8 +162,15 @@ func (r VDR) Update(id did.DID, current hash.SHA256Hash, next did.Document, _ *t
 		return err
 	}
 
-	keyID := controllers[0].CapabilityInvocation[0].ID.String()
-	_, err = r.network.CreateTransaction(didDocumentType, payload, keyID, nil, time.Now(), currentMeta.SourceTransactions)
+	key, err := r.keyStore.Resolve(controllers[0].CapabilityInvocation[0].ID.String())
```

Comment:
is the first controller guaranteed to have a `capabilityInvocation` key? Otherwise this'll lead to an out of bounds error

Reply:
nope, this should be fixed by the `canControl` functionality that was started on. I'll make sure there's an issue for that.

Reply:
#266